### DISCUSSION TITLE
ViaATV [MQ]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#363] [Show and tell]
### Channel ID

ViaATV.mq

### Stream URL

https://streamer01.myvideoplace.tv/streamer02/hls/ATV_DIRECT_EV_111018.m3u8

### Notes

Not 24-7

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Will have to verify for this iteration but the "vià' list of channels is originally from France.

See https://fr.wikipedia.org/wiki/Vi%C3%A0ATV

**Comment by freearhey** [comment]
https://streamer01.myvideoplace.tv/streamer02/hls/ATV_DIRECT_EV_111018.m3u8 (offline)
<img width="568" alt="image" src="https://user-images.githubusercontent.com/7253922/232244768-08bb7e66-d1fa-4f17-b25a-dcc23c961861.png">
